tests/tck-build-logic/src/main/groovy/org.graalvm.internal.tck.gradle

```diff
@@ -20,10 +20,8 @@ repositories {
     mavenCentral()
 }
 
-java {
-    toolchain {
-        languageVersion = JavaLanguageVersion.of(11)
-    }
+tasks.withType(JavaCompile) {
```

Comment:
See commit message of https://github.com/oracle/graalvm-reachability-metadata/pull/37/commits/05794064a6d792612102f4ae6800408838c37294 for details.